homeassistant/components/homewizard/coordinator.py

```diff
@@ -74,8 +74,10 @@ async def _async_update_data(self) -> DeviceResponseEntry:
 
                 # Do not reload when performing first refresh
                 if self.data is not None:
-                    await self.hass.config_entries.async_reload(
-                        self.config_entry.entry_id
+                    self.hass.async_create_task(
+                        self.hass.config_entries.async_reload(
+                            self.config_entry.entry_id
+                        )
```

Comment:
There is a helper for this:
```suggestion
                    self.hass.config_entries.async_schedule_reload(
                        self.config_entry.entry_id
                    )
```

You should also try to add tests